index.html

```diff
@@ -415,9 +415,9 @@
             },
           ],
           antwoorden: [
-            "Ik voel me meestal goed en ik kan mezelf zijn. Veel mensen (bv. mijn ouders, mijn vrienden, de juf, …  ) zien mij graag en ik zie hen ook graag. Dit tonen we aan elkaar.",
-            "Ik voel mij best OK. De ene keer gaat het al wat beter dan de andere keer. Waar ik echt goed in ben weet ik nog niet zo zeker. Als ik iets doe of iets maak (op school, van mijn ouders, van vrienden) krijg ik wel eens complementjes, soms weet ik niet of die wel gemeend zijn.",
             "Mijn gevoelens zijn iets van mezelf, daar loop ik niet mee te koop. Ik probeer dingen te doen zodat anderen (bv. mijn ouders, mijn vrienden, de juf, …  ) mij leuk vinden. Wanneer ik iemand graag heb dan weet ik niet goed hoe je dat kunt tonen. Ik ben soms wat onzeker. Mensen zeggen mij waar ik niet zo goed in ben, ik zou het leuk vinden af en toe te horen waar ik goed in ben.",
```

Comment:
deze is niet gewoon omgedraaid, is dat opzettelijk?

Reply:
laag: mijn gevoelens...
midden: ik voel mij best OK
hoog: ik voel me meestal goed en ...

Reply:
maar hoog staat altijd eerst denk ik